language/functional_tests/src/evaluator.rs

```diff
@@ -81,44 +82,46 @@ impl OutputType {
     }
 }
 
-/// An entry in the `EvaluationResult`.
+/// An entry in the `EvaluationLog`.
 #[derive(Debug)]
 pub enum EvaluationOutput {
-    Transaction,
+    Transaction(usize),
```

Comment:
Nit: comment describing what the `usize` means? I believe Rust also lets you do this using an inline enum (e.g., `Transaction { status_code: usize }`)

Reply:
It's the index of the transaction. I'll leave a comment there. (Feel like `Transaction { id: usize }` is a little bit heavyweight. Thoughts?)